csrankings-a.csv

```diff
@@ -2146,6 +2147,7 @@ Arvind Narayanan,Princeton University,http://randomwalker.info,0Bi5CMgAAAAJ
 Arwa A. Jamjoom,King Abdulaziz University,http://www.kau.edu.sa/Contacts.aspx?Site_ID=611&Lng=EN&Cont=9002,yMcPOd0AAAAJ
 Arya Mazumdar,Univ. of California - San Diego,http://mazumdar.ucsd.edu,9tjQU1EAAAAJ
 Aryabartta Sahu,IIT Guwahati,https://www.iitg.ernet.in/asahu,apzckLUAAAAJ
+Arye Nehori,Washington University in St. Louis,https://www.ese.wustl.edu/~nehorai/,0pp-L_IAAAAJ

```

Comment:
Not in CS, please clarify.

Reply:
He has a courtesy appointment in CSE: https://cse.wustl.edu/faculty-research/courtesy-appointments.html. As does Albert M. Lai, also added in this pull. 

My reading of the guidelines for inclusion seemed to be satisfied by the departmental rules: https://cse.wustl.edu/documents/CSE-Doctoral-Program-Guide.pdf.